docs/pages/components/Panel.mdx

```diff
@@ -0,0 +1,261 @@
+---
```

Comment:
I was actually wondering about this - do we actually need the collapsed variant any longer? I ask, because the padding on the top element is already `12px`, and collapsed is only taking things from `16px` -> `12px`. I'll add it back in either way, but was something meant to check with you on.

Reply:
Let me check with @bobbyomari and @shawnsharpDQ, it might not be needed but of course we'd still have to deprecate it if not.